test/cloud_firestore_mocks_test.dart

```diff
@@ -319,7 +319,7 @@ void main() {
     expect(snapshot1, isNotNull);
     expect(snapshot1.id, nonExistentId);
     // data field should be null before the document is saved
```

Comment:
This unit test was introduced in #32 to match Firestore's behavior. It turns out that if you query a non-existent document, when you retrieve its `data()`, Firestore returns null. So we should continue to return null here.